packages/kubernetes-1.15/kubelet-config

```diff
@@ -29,6 +29,20 @@ authorization:
 clusterDomain: {{settings.kubernetes.cluster-domain}}
 clusterDNS:
 - {{settings.kubernetes.cluster-dns-ip}}
+{{~#if settings.kubernetes.eviction-hard}}
+evictionHard:
```

Comment:
This is awesome - we've built an AMI with this patch and its working wonderfully. In a followup PR, we'd like to see the `evictionSoft` settings exposed, as well as the `eviction-max-pod-grace-period` setting. Overall though, this is great!

Reply:
Hi @diranged. We have opened a new issue about supporting `eviction soft` related settings. You can follow up here https://github.com/bottlerocket-os/bottlerocket/issues/1445. Thanks!